File: Lab 3/Task3.rs
fn main() {
    println!("Input 2 integers:");

    let mut buf = String::new();
    std::io::stdin().read_line(&mut buf).unwrap();
    let mut recv = buf.split_whitespace();
    let a = recv.next().unwrap().parse::<i32>().unwrap();
    let b = recv.next().unwrap().parse::<i32>().unwrap();

    println!("The bigger one is {}", if a >= b {a} else {b});
}

File: Lab 3/Task3.lua
print("Input 2 integers:")
local buf_iter = string.gmatch(io.read(), "%d+")
local a, b = tonumber(buf_iter()), tonumber(buf_iter())

-- "Lua does not have a traditional conditional operator. However, the
-- short-circuit behaviour of its "and" and "or" operators allows the emulation
-- of this behaviour"
--   https://en.wikipedia.org/wiki/%3F:#Lua
-- See also §3.4.5
print(string.format("The bigger one is %d", a >= b and a or b))
